### ISSUE TITLE
lines3D tooltip

### ISSUE THREAD

**Issue by jarben** [ecomfe/echarts-gl#52]
Would be great if lines3D supports tooltip:)

**Comment by helgasoft** [comment]
+1

**Comment by creepyao** [comment]
Would be great if lines3D supports tooltip

**Comment by helgasoft** [comment]
yes, or have an option to activate tooltip, like the [triggerLineEvent](https://echarts.apache.org/en/option.html#series-line.triggerLineEvent) in 2D.
